prisma/seeds/prices.ts

```diff
@@ -0,0 +1,35 @@
+import { Prisma } from '@prisma/client'
+export const prices = [
+  { // XECUSD
+    value: new Prisma.Decimal('0.00004095051166'),
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    timestamp: 166459320,
+    networkId: 1,
+    quoteId: 1
+  },
+  { // XECBTC
```

Comment:
Half of the team is Canadian and we do sort of need CAD conversions (which the API does have rn) so I'd rather we just do that instead of BTC here and below.

Reply:
Addressed this and the other remarks (check last commit)